friends/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import axios from 'axios';
+import './App.css';
+import FriendsList from './FriendsList';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: [],
+      newFriend: ''
+    }
+  }
+
+componentDidMount() {
+  axios.get('http://localhost:5000/friends').then(response => {
+    console.log(response);
+    this.setState({friends: response.data})
+  })
+  .catch(err => {
+    console.log(err)
+  })
+}
+
+handleNameChange = e => {
+  this.setState({newFriend: e.target.value})
+}
+
+handleNewFriend = () => {
+  const name = {name: this.state.newFriend}
+  axios.post('http://localhost:5000/friends', name)
```

Comment:
what if we want to add the age and email of our new friend? how will we contact them?!